recipe/meta.yaml

```diff
@@ -12,7 +12,7 @@ source:
     - 0001_use-setuptools.patch
 
 build:
-  number: 0
+  number: 2
```

Comment:
I was of the impression that if you rerender without version change you are supposed to increase the build number. A jump by 2 was just a mistake. I have corrected it in the 43c03a2.